railties/CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 
 *   Don't enable YJIT in development and test environments
 
-    Development and test environment tend to reload code and redefine methods (e.g. mocking),
+    Development and test environments tend to reload code and redefine methods (e.g. mocking),
     hence YJIT isn't generally faster in these environments.
```

Comment:
Can you fix here as well?

https://github.com/rails/rails/blob/3fa41c023de525f8a067ba61415311f9daf5b575/railties/lib/rails/application/configuration.rb#L352-L355

Reply:
Done!